src/pages/special/panel/AdminPanel.jsx

```diff
@@ -22,13 +22,78 @@ import AdminEdit from "./../edit/Edit";
 import * as C from "@utils/constants";
 import * as S from "./style";
 
+const cmsNavData = [
+  {
+    title: "Admin Panel",
+    path: "/panel",
+    authFor: [
+      C.userConstants.USER_STATUS_CREW,
+      C.userConstants.USER_STATUS_DEVELOPER,
+    ],
+  },
+  {
+    title: "Add new content",
+    authFor: [C.userConstants.USER_STATUS_DEVELOPER],
+    content: [
+      {
+        title: "Add event",
+        path: "/panel/add/events",
+      },
+      {
+        title: "Add article",
+        path: "/panel/add/article",
+      },
+      {
+        title: "Add news",
+        path: "/panel/add/news-content",
+      },
+    ],
+  },
+  {
+    title: "Manage content",
+    authFor: [C.userConstants.USER_STATUS_DEVELOPER],
+    content: [
+      {
+        title: "Manage events",
+        path: "/panel/manage/events",
+      },
+      {
+        title: "Manage article",
+        path: "/panel/manage/article",
+      },
+      {
+        title: "Manage news",
+        path: "/panel/manage/news-content",
+      },
+    ],
+  },
+  {
+    title: "Translate content",
+    authFor: [C.userConstants.USER_STATUS_DEVELOPER],
+    content: [
+      {
+        title: "Translate footer",
+        path: "/panel/translate/footer",
+      },
+    ]
+  },
+  {
+    title: "Manage your profile",
+    path: "/panel",
+    authFor: [C.userConstants.USER_STATUS_DEVELOPER]
+  },
+  {
+    title: "Manage pictures",
+    path: "/panel",
+    authFor: [C.userConstants.USER_STATUS_DEVELOPER]
+  }
+];
+
 const AdminPanel = () => {
   const [isPanelNavOpen, setIsPanelNavOpen] = useState(false);
 
   const alert = useSelector((state) => state.CMS.alert);
-  const userStatus =
-    useSelector((state) => state.currentUser.status) ||
-    C.userConstants.USER_STATUS_DEVELOPER;
+  const userStatus = C.userConstants.USER_STATUS_DEVELOPER;
```

Comment:
ja bym zrobił funkcje, która sprawdza statusy i mu odpowiednim nadaje uprawnienia :P raz, że robisz tę funkcję tylko dwa i nie musisz powtwarzać kodu a dwa, wygląda to czytelniej :D